.github/workflows/ci.yml

```diff
@@ -0,0 +1,104 @@
+name: CI
+
+on: [
+    push,
+    pull_request
+]
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.4',
```

Comment:
Best to add 7.3 here

Reply:
7.3 is covered in the prefer-lowest run

Reply:
Thats not enough though as that doesnt check the latest versions of dependencies against our code base in 7.3
So best to add that as separate run here.